cli/flox/src/commands/pull.rs

```diff
@@ -297,7 +336,44 @@ impl Pull {
                 query_ignore_build_errors: Self::query_ignore_build_errors,
                 query_migrate_and_add_system: Self::query_migrate_and_add_system,
             }),
-        )
+        )?;
+
+        if copy {
+            debug!("Converting environment to path environment");
+            if let Err(e) = env.into_path_environment(flox) {
+                fs::remove_dir_all(&dot_flox_path)
+                    .context("Could not clean up .flox/ directory")?;
+                bail!(e);
+            }
+        }
+
+        let message_lead = if copy {
+            format!("Created path environment from {env_ref}.")
+        } else {
+            format!(
+                "Pulled {env_ref} from {floxhub_host}.",
+                floxhub_host = flox.floxhub.base_url()
+            )
```

Comment:
note nonblocking: if it would simplify things, I'm sure it would be fine to print `Pulled {env_ref} from {floxhub_host}."` and then `Created path environment from {env_ref}.` I don't think a user would care that we printed that info in two steps

Reply:
Well, now that its factored out i guess we can have that part conditional.

The issue was that we'd otherwise print

```
✨ Pulled {env_ref} from {floxhub_host}.

You can activate the environment with 'flox activate'
✨ Created path environment from {env_ref}.
```


its that context in the middle that needs to be deferred at least if we go by the styleguide and require context to be printed last.